locales-pending/fix.ftl

```diff
@@ -0,0 +1,63 @@
+fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = View your info on these sites
+fix-flow-data-broker-profiles-view-data-broker-profiles-view-profile = View your profile
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-view-more = View more  
+fix-flow-data-broker-profiles-view-data-broker-profiles-button-view-less = View less  
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-headline = Here’s how to remove these profiles 
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-title = Visit each site and find removal instructions
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-1-content = They may be called data suppression, delete my data, do not show my data, or opt out instructions.
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-2-title = Follow the steps outlined on the site
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-2-content = Some sites may allow you to fill out an opt-out form online, while others may require you tomail a letter. 
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-3-title = Mark as fixed on this page
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-3-content = This will help you keep track of which exposures still need your attention, and make youroverall exposures go down.
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-4-title = Watch for re-exposures
+fix-flow-data-broker-profiles-manual-remove-how-to-remove-step-4-content = This process does not remove your information from public records, so even after your information is removed, sites often add you back. If you’d like us to handle this for you and automatically remove any new exposures, upgrade to Monitor Premium.
+fix-flow-data-broker-profiles-manual-remove-review-profiles-headline = Review & remove your profiles
+fix-flow-data-broker-profiles-manual-remove-button-remove-for-me = Remove them for me
+fix-flow-data-broker-profiles-manual-remove-button-skip = Skip for now 
+fix-flow-data-broker-profiles-automatic-remove-headline = We’ll auto-remove these and guide you through exposures that need manual fixes
+# Variables:
+# $data_broker_count is the number of data brokers scanned monthly (As of adding this, it is 190). This number will always be plural.
+fix-flow-data-broker-profiles-automatic-remove-subheadline = Removing these stops the sale of your personal info on { $data_broker_count } sites. Upgrade to Monitor Premium and you’ll get:
+# Variables:
+# $percent is the percentage you can save subscribing to an annual/yearly plan
+fix-flow-data-broker-profiles-automatic-remove-save-percent = Save { $percent }% with yearly plan
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-headline = Premium Protection
+fix-flow-data-broker-profiles-automatic-remove-features-select-plan-toggle-yearly =  Yearly
```

Comment:
Only one space before and after the equal sign.

Reply:
Removed in 6e660ee1c87da0f3c7f9648f7c9868b698bbf957